Classes/DataProcessor.php

```diff
@@ -0,0 +1,45 @@
+<?php
+declare(strict_types=1);
+
+namespace PLUSWERK\Dashboard;

```

Comment:
```suggestion
namespace Pluswerk\Dashboard;
```
that is our vendor-name for pluswerk projects